### PR TITLE
Allow translation of UCR report filters

### PR DESCRIPTION
**Pull request by NoahCarnahan**
Allows users building UCR reports to specify translations for report filter labels. Works just like column heading translation introduced here: https://github.com/dimagi/commcare-hq/pull/7016. Can be reviewed commit-by-commit.

cc @gcapalbo

### CODE REVIEW

File: corehq/apps/userreports/util.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    Localize the given value.
+
+    This function is intended to be used within UCR to localize user supplied
+    translations.
+
+    :param value: A dict-like object or string
+    :param lang: A language code.
+    """
+    if isinstance(value, basestring) or value is None:
+        return value
```
Comment (gcapalbo) [inline corehq/apps/userreports/util.py]: are these two lines even necessary?

Comment (NoahCarnahan) [inline corehq/apps/userreports/util.py]: Ah right. I guess not. Fixing...

**Comment by gcapalbo** [comment]
looks like it just needs to be merged with master (can't be automatically merged). otherwise, is good to merge

**Comment by NoahCarnahan** [comment]
Fixed, thanks.

**Pull request merged**